python/lsst/analysis/tools/actions/scalar/scalarActions.py

```diff
@@ -246,3 +246,90 @@ def __call__(self, data: KeyedData, **kwargs) -> Scalar:
         mask = self.getMask(**kwargs)
         arr = cast(Vector, data[self.vectorKey.format(**kwargs)])[mask]
         return cast(Scalar, np.nansum(arr))
+
+
+class MedianHistAction(ScalarAction):
```

Comment:
I think you also need to add `MedianHistAction` and `IqrHistAction` to the `__all__`.

Reply:
Done by @aemerywatkins.